catalog/app/utils/Config.js

```diff
@@ -18,32 +18,37 @@ const parseJSON = (msg = 'invalid JSON') =>
     throw new ConfigError(`${msg}:\n${src}`, { src, originalError: e })
   })
 
+const SSO_PROVIDERS = ['google']
+
 const validateConfig = conforms({
   registryUrl: R.is(String),
   alwaysRequiresAuth: R.is(Boolean),
-  sentryDSN: isNullable(String),
-  intercomAppId: isNullable(String),
-  mixpanelToken: isNullable(String),
+  sentryDSN: isNullable(R.is(String)),
+  intercomAppId: isNullable(R.is(String)),
+  mixpanelToken: isNullable(R.is(String)),
   apiGatewayEndpoint: R.is(String),
   defaultBucket: R.is(String),
   signInRedirect: R.is(String),
   signOutRedirect: R.is(String),
-  disableSignUp: isNullable(Boolean),
+  passwordAuth: isNullable(oneOf([true, false, 'SIGN_IN_ONLY'])),
```

Comment:
Reminds me of [this](https://thedailywtf.com/articles/What_Is_Truth_0x3f_) :)

Reply:
:) i can make this a little more descriptive like `isNullable(oneOf(['SIGN_UP', 'SIGN_IN_ONLY']))` (with `null` meaning disabled) or maybe `[false, 'SIGN_UP', 'SIGN_IN_ONLY']`